shopify_app.gemspec

```diff
@@ -8,9 +8,7 @@ Gem::Specification.new do |s|
   s.author      = "Shopify"
   s.summary     = %q{This gem is used to get quickly started with the Shopify API}
 
-  s.add_dependency('rails', '>= 4.2.6', '< 5.0')
```

Comment:
@andriytyurnikov how about this, change this PR to *only* address the direct problem. The fix:

```diff
- s.add_dependency('rails', '>= 4.2.6', '< 5.0')
+ s.add_dependency('rails', '>= 4.2.6')
```

After that we'll update the CI suite and matrix to our needs and coordinate new releases.